content/en/security/application_security/code_security/setup/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Code Security Setup
+disable_toc: false
+further_reading:
+- link: "logs/processing/pipelines"
+  tag: "Documentation"
+  text: "Log processing pipelines"
+- link: "/security/application_security/code_security"
+  tag: "Documentation"
+  text: "Code Security"
+- link: "https://www.datadoghq.com/blog/iast-datadog-code-security/"
+  tag: "Blog"
+  text: "Enhance application security in production with Datadog Code Security"
+- link: "https://www.datadoghq.com/blog/code-security-owasp-benchmark/"
+  tag: "Blog"
+  text: "Datadog Code Security achieves 100 percent accuracy in OWASP Benchmark by using an IAST approach"
+---
+
+## Prerequisites
+Before setting up Code Security, ensure the following prerequisites are met:
+
+1. **Datadog Agent Installation:** The Datadog Agent is installed and configured for your application's operating system or container, cloud, or virtual environment.
+2. **Datadog APM Configuration:** Datadog APM is configured for your application or service, and traces are being received by Datadog.
+3. **Supported Tracing Library:** The Datadog Tracing Library used by your application or service supports Software Composition Analysis capabilities for the language of your application or service. For more details, refer to the [Library Compatibility][1] page.
+
+## Code Security Enablement Types
+
+There are two main approaches to enable Code Security on your tracing libraries: Single-Step Instrumentation and Datadog Tracing Libraries.
+
+### Single-Step Instrumentation
+
+Run a one-line install command to install the Datadog Agent, and enable Code Security with [Single-Step Instrumentation][2].
```

Comment:
```suggestion
Run a one-line install command to install the Datadog Agent, and enable Code Security with [Single-Step Instrumentation][2].
```
```suggestion
Run a one-line install command to install the Datadog Agent, and enable Code Security with [Single-Step Instrumentation][2].
```
this link just links back to the same page, where should it go?

Reply:
fixed. I incorporated the entire file into the main setup. So it's a one-pager now.
You will see it in my next commit.